maat-court-data-api/src/main/java/gov/uk/courtdata/entity/RepOrderEntity.java

```diff
@@ -51,6 +53,11 @@ public class RepOrderEntity {
     @OneToMany(mappedBy = "repOrder", fetch = FetchType.LAZY, cascade = CascadeType.ALL, orphanRemoval = true)
     private final List<IOJAppealEntity> iojAppeal = new ArrayList<>();
 
+    @ToString.Exclude
+    @JsonIgnore
+    @OneToMany(mappedBy = "repOrder", fetch = FetchType.LAZY, cascade = CascadeType.REMOVE, orphanRemoval = true)
```

Comment:
What does this cascade type do?
Also, why have we got this JsonIgnore and the other don't?

Reply:
The cascade ensures the RepOrderHistory is deleted for this RepOrder when the RepOrder is deleted.

I've removed the JsonIgnore - I forgot to remove it after earlier issues and experimentation :)

Reply:
Thanks!